.github/workflows/auto-diff-k8s-ci.yaml

```diff
@@ -108,13 +111,62 @@ jobs:
       submit: false
     secrets: inherit
 
+  check_kind_node_version:
```

Comment:
// update k8s matrix in step "call_k8s_matrix"
update_k8s_matrix

Reply:
done